api/api.go

```diff
@@ -48,6 +51,21 @@ type Authority interface {
 	Version() authority.Version
 }
 
+type LinkedAuthority interface { // TODO(hs): name is not great; it is related to LinkedCA, though
```

Comment:
I think it might make more sense to name this something associated with Admin (not linked). Everything is kinda related to linkedCA, because everything has to work as a linkedCA as well.

Maybe `AdminAuthority` or something with the word admin, since it is the admin API.